cdap-client-tests/src/test/java/co/cask/cdap/client/MetadataHttpHandlerTestRun.java

```diff
@@ -1020,14 +967,6 @@ public void testInvalidSearchParams() throws Exception {
     } catch (BadRequestException e) {
       // expected
     }
-
-    // search with invalid query
-    try {
-      searchMetadata(NamespaceId.DEFAULT, "");
```

Comment:
Because it is no longer invalid. The existing code has "" as the default in the REST API, but that never worked. So i changed it mean the same as "*".